src/options.js

```diff
@@ -8,15 +9,22 @@ export const optionDefaults = {
 const optionMap = {
   input: {
     buildParams: (filter, json, params, value) => {
-      if (value === 'file') {
-        validateJSONPath(params[params.length - 1])
-      } else {
-        params.pop()
-        params.unshift('--null-input')
-        if (value === 'json') {
-          json = JSON.stringify(json)
-        }
-        params[params.length - 1] = `${json} | ${filter}`
+      switch (value) {
+        case 'file':
+          validateJSONPath(params[params.length - 1])
+          break
+
+        default:
+          params.pop()
+          params.unshift('--null-input')
+          if (value === 'json') {
+            json = JSON.stringify(json)
+          }
+          if (value === 'json5') {
```

Comment:
Instead of having separate inputs for `json` and `json5`, how about we allow the input to be in `json5` format, since that's just a leaner version? Then, return output always as regular `json`.